src/components/TryItNow.js

```diff
@@ -44,12 +44,9 @@ export const TryItNow = ({ requiresKeys }) => {
 						break;
 					}
 					if (!input) input = await window.prompt(match)
-					if (!input && !allowNoInput.includes(match)) throw 'return'
+					if (!input && !allowNoInput.includes(match)) return;
```

Comment:
We should have some sort of error here right?

When the user doesn't input anything and it just "stops" the fetch call because there was no input. 

I realize this was left over by me, but would you mind including something, testing it out and then let's merge?

Reply:
sure, will add!